bin/kano-start-browser

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+USER_AGENT='Mozilla/5.0 (iPhone; CPU iPhone OS 6_0 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/6.0 Mobile/10A5376e Safari/8536.25'
+chromium-browser --start-maximized --user-agent="$USER_AGENT"
```

Comment:
sorry to come in late Alex @Ealdwulf , if we prepend the command with this snippet, we'll get a nice startup hourglass: `kdesk-hourglass-app chromium-browser ; blabla`